crates/biome_js_analyze/tests/specs/nursery/noStaticElementInteractions/invalid.jsx

```diff
@@ -23,7 +23,6 @@
 	<hgroup onClick={() => {}} />
 	<i onClick={() => {}} />
 	<kbd onClick={() => {}} />
-	<link onClick={() => {}} href="#" />
```

Comment:
This test was [recently commented (and thus removed) in the test suite of the equivalent ESLint rule](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/__tests__/src/rules/no-static-element-interactions-test.js#L88)